hw7/ex34.py

```diff
@@ -0,0 +1,9 @@
+def fun(words):
+    return sum(1 for i in words if i in 'аеёиоуыэюя')
```

Comment:
Можно иначе
```suggestion
    return sum(i in 'аеёиоуыэюя' for i in words)
```
Подумайте, что я сделал и как это работает.